oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+GitHub Groups
+^^^^^^^^^^^^^
+
+As a companion to the Decision Maker's Checklist, a set of GitHub groups will be created in the ``openedx`` organization that can be tagged on Pull Requests for ADRs.  The Decision Maker's Checklist will indicate the GitHub handles of individuals, or of these groups for use in creating pull requests.
+
+Checklist Organization
+^^^^^^^^^^^^^^^^^^^^^^
+
+The Decision Maker's Checklist is a reference, and so decision maker's will be able to look up individuals by various criteria.  To this end, the checklist will list stakeholders and experts for products, domains, and by role, at a minimum.  This means that individuals or groups may appear more than once if they're experts on several different domains, or have multiple roles, for instance.
+
+We will necessarily iterate on the organization of the checklist as we learn more about how we use it.
+
+Technical Strategic Review (WIP)
+--------------------------------
+
+Coinciding with an organization's broader product strategy and prioritization cadence, establish quarterly technical strategic reviews focused on engineering and architectural priorities as they relate to our businesses' and platform's needs.  Depending on the size and team topologies of a given organization, The number of such meetings may vary.
+
+There may several reviews, and they may be domain/theme-specific depending on the organization.  This process is not prescriptive on the number or cadence of review ceremonies, except to say that they should happen as frequently as an organization's business/product planning process occurs.
+
+The review should be attended by engineering managers and at least one technical lead from each team in the group.
+
+Strategic Review Artifacts (WIP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Can we create a consistent set of artifacts to come out of technical strategic review that can be shared as input into downstream discussions (roadmap discussions, *other* strategic reviews, etc.)?  What should that artifact look like?
+
+Proposal: A user story roadmap, such as: `Architecture User Story Roadmap <https://openedx.atlassian.net/wiki/spaces/AC/pages/1812037671/FY21-22+Architecture+User+Story+Roadmap>`_ from FY21-22.
+
+
+Other Options Considered
+========================
+
+Require Engineers at Product Ceremonies
+---------------------------------------
+
+What if we just invite engineers to product planning ceremonies so they can get involved earlier?
+
+Simply put, these are the wrong forums for the discussions engineers need to have.  Certainly some technical details are brought up, but they're not the focus, and the meeting doesn't have the right attendance to get into the details.
+
+A review of notes from a recent product strategy meeting at 2U lends some credibility to this sentiment - the focus is on business-focused strategies and plans, not engineering-focused initiatives to support them.
```

Comment:
Another reason is that currently these are 2U centric meetings and not community centric and so it still doesn't solve for a process that is open to the entire open edx community.

Reply:
I've removed this section completely.